src/shared/pages/list/mpamEnquiryReasons/enquirySubjectView.tsx

```diff
@@ -65,6 +70,9 @@ const EnquirySubjectView: React.FC<CasesProps> = ({ history, match }) => {
                                             return (
                                                 <tr className='govuk-table__row' key={enquirySubject.simpleName}>
                                                     <td className='govuk-table__cell'>{enquirySubject.title}</td>
+                                                    <td className="govuk-table__cell">
+                                                        <a href="#" onClick={event => onAmendEnquiryReasonClick(history, enquirySubject.uuid, subject)}>Amend</a>
```

Comment:
Should be able to use the <Link> component here which uses react-router-dom for the routing rather than doing manual history pushes in the component.

See header.tsx line 12 for usage example